Crash when executing on a repo

**Which version of kube-score are you using?**
kube-score version: 1.7.0, commit: 37cddd6eaff9ba3af91fdafe5c0dda83db009a20, built: 2020-05-14T18:06:56Z
**What did you do?**

Executed `$ kube-score`

**What did you expect to see?**
No error



**What did you see instead?**

``` bash
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
main.parse(0xc0001021b0, 0x1, 0x1, 0xc000167e78, 0xc000167f48, 0x1, 0x30, 0xc000220e40, 0x1)
	/Users/gustav/src/kube-score/cmd/kube-score/cmd.go:28 +0x20a
main.main()
	/Users/gustav/src/kube-score/cmd/kube-score/main.go:52 +0x2f9
```

--- a/cmd/kube-score/cmd.go
+++ b/cmd/kube-score/cmd.go
@@ -25,9 +25,11 @@ func parse(args []string, cmds map[string]cmdFunc) (command string, cmdArgsOffse
 	}
 
 	// If arg 1 is set and is a valid command, always use it as the command to execute, instead of the default
-	if _, ok := cmds[args[1]]; ok {
-		command = args[1]
-		cmdArgsOffset = 2
+	if len(args) > 1 {
+		if _, ok := cmds[args[1]]; ok {
+			command = args[1]
+			cmdArgsOffset = 2
+		}
 	}
 	return
 }

